docs/src/main/sphinx/connector/sql-update-limitation.fragment

```diff
@@ -0,0 +1,29 @@
+### UPDATE
+
+Only ``UPDATE`` statements with constant assignments and predicates are
+supported. For example, the following statement is supported:
+
+```sql
+UPDATE table SET col1 = 1 WHERE col3 = 1
+```
+
+The following statement is not supported because arithmetic expressions cannot
```

Comment:
what about function calls or any other value derived from other columns in the same row? Or any other expressions besides arithmetic?

Reply:
> what about function calls or any other value derived from other columns in the same row? Or any other expressions besides arithmetic?

All cases you mentioned - not supported, only kind of constant updates